spec/javascripts/utilities/ReCaptchaV3.spec.tsx

```diff
@@ -0,0 +1,29 @@
+import { mount } from 'enzyme'
+
+import type { Props } from 'utilities/ReCaptchaV3'
+
+import type { FunctionComponent } from 'react'
+
+const ReCaptchaV3 = jest.requireActual('utilities/ReCaptchaV3').ReCaptchaV3 as FunctionComponent<Props>
+
+describe('ReCaptchaV3', () => {
+  it('should render itself correctly', () => {
+    const wrapper = mount(<ReCaptchaV3 enabled action="test/action" siteKey="fakeKey" />)
```

Comment:
This component loads a JS script from google. @josemigallas do you know whether this test would actually download the script every time it runs?

Reply:
Surprisingly I did exactly that and the cuke doesn't fail: https://github.com/3scale/porta/blob/1cbf07e991bec976bb8b9a753c5cda740410be85/features/provider/login.feature

Reply:
We probably mock kit somewhere, somehow. For javascript tests you can create a mock like like the ones for Braintree and Stripe: https://github.com/3scale/porta/blob/a25da8553ddd1b1f20f85a8a8e3988c299596f8f/spec/javascripts/__mocks__/braintree-web.js